inc/settings.php

```diff
@@ -125,7 +125,7 @@ function siteorigin_north_settings_init(){
 				'sidebar_width' => array(
 					'label'             => __( 'Sidebar Width', 'siteorigin-north' ),
 					'type'              => 'measurement',
-					'live'              => true,
+					'live'              => false,
```

Comment:
That's actually a much better solution. Will something like `overflow-x: hidden` fix this? That seems ideal, although we might run into problems with some widgets that expect to be able to overflow. For example, a widget that displays image thumbnails that displays a larger preview when you hover.

Does this issue only happen at `0`? If so, don't all measurements have a similar interpretation of the value `0`? So `0px == 0% == 0em == 0`?